### DISCUSSION TITLE
Copilot summaries for Discussions: Private Preview Feedback

### DISCUSSION THREAD

**Discussion by rileybroughten** [community/community#141943] [Copilot Conversations] [answered]
Thank you for participating in the preview of Copilot summaries for Discussions, allowing you to quickly understand what a discussion is about and the key information you need to know, saving you time to gather insights and next steps. 

If you'd like to join the preview, please add a comment to enable this for your username or organization(s).

### How it works

You can select `Summarize` from the top of a discussion, which will generate a summary of the discussion and comments. You can use the `Follow up in chat` option to continue the conversation with Copilot chat to ask additional questions.

https://github.com/user-attachments/assets/c6521baa-d79f-4238-8c93-ad5564c0db17

The summary only appears for the user that generated it rather than all users, and it will persist when revisiting or refreshing the page.

### Looking for feedback

You can let us know your feedback using the :thumbsup: / :thumbsdown: options in the banner to provide feedback.

Let us know what you think by providing details on when and how you use discussion summaries, what is working well, where we can improve, and other things you'd like to see in a comment below 👇

**Comment by kkamegawa** [comment]
Does Summarize feature always generating in English? 
For example, this issue only written in Japanese, but summarize generate in English.

https://github.com/Implem/Implem.Pleasanter/issues/545

If issues in Japanese are summarized in Japanese, it will be easier to catch up when using local languages in internal repositories, etc.

**Comment by rileybroughten** [comment, marked as answer]
👋 We wanted to provide an update that we are not moving forward with this experience and have rolled it back for those that had access. We are continuing to iterate on and improve this experience to understand summaries, and will keep you all updated on our next steps in this space. We appreciate all of the feedback you provided!

We encourage you to check out [GitHub’s public roadmap](https://github.com/orgs/github/projects/4247/views/1) to stay tuned on upcoming releases!
